src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -42,6 +42,16 @@ public Intake()
 
   // METHODS //
 
+  /* =====================================
+  flip() deploys the intake if it is 
+  retracted and retracts the intake if 
+  it is deployed
+  ===================================== */
+  public void flip() {
+    if (deployed) retract();
+    else deploy();
+  }
```

Comment:
It's always better to put in brackets in case you need to add more lines of code later:
```
if (condition)
{
    // whatever
}
else
{
   // whatever
}
```
Or you could do this depending on your style:
```
if (condition) {
    // whatever
} else {
   // whatever
}
```